v3/tasks/20250221-v3-cow-swap-fee-burner/input.ts

```diff
@@ -0,0 +1,44 @@
+import { Task, TaskMode } from '@src';
+
+export type CowSwapFeeBurnerDeployment = {
+  Vault: string;
+  AppDataHash: string;
+  ComposableCow: string;
+  CowVaultRelayer: string;
+  Version: string;
+};
+
+const Vault = new Task('20241204-v3-vault', TaskMode.READ_ONLY);
+
+/*
+NOTE: AppDataHash was encoded here: https://explorer.cow.fi/appdata?tab=encode
+
+AppData:
+{
+  "appCode": "{\"name\":\"CowSwapFeeBurner\",\"version\":1,\"deployment\":\"20250221-v3-cow-swap-fee-burner\"}",
+  "metadata": {
+    "hooks": {
+      "version": "0.1.0"
+    }
+  },
+  "version": "1.3.0"
+}
+*/
+const AppDataHash = '0xc800f61d18fc71c6da4c62377e0b2e94eddde7f5bdcd978d0aa501098e30f427';
```

Comment:
I couldn't reproduce this AppDataHash from the site. Maybe whitespace or \"? Or did other data get entered? The decode on the site doesn't seem to work for me either (just get the hourglass forever).

Reply:
Works for me.

Just enter this in the text box for `AppCode`:

```
{"name":"CowSwapFeeBurner","version":1,"deployment":"20250221-v3-cow-swap-fee-burner"}
```

Reply:
Yes; I was trying to do the whole thing.

Reply:
the json actually has to be uploaded to ipfs in order for the appdatahash to be used to retrieve said json. a hash can never be 'reversed engineered' back to its content; hashing is one directional.

i tried uploading it myself via [cow's endpoint](https://api.cow.fi/docs/#/default/put_api_v1_app_data__app_data_hash_) but im afraid the nested json might be causing problems (not sure; but i keep getting `400: {"errorType": "AppDataInvalid","description": "invalid app data json"}` back)

ref: https://github.com/cowprotocol/services/issues/2302